db/db_impl/db_impl_write.cc

```diff
@@ -1338,12 +1340,14 @@ Status DBImpl::PreprocessWrite(const WriteOptions& write_options,
   // It does soft checking because WriteBufferManager::buffer_limit_ has already
   // exceeded at this point so no new write (including current one) will go
   // through until memory usage is decreased.
-  if (UNLIKELY(status.ok() && write_buffer_manager_->ShouldStall())) {
-    if (write_options.no_slowdown) {
-      status = Status::Incomplete("Write stall");
-    } else {
-      InstrumentedMutexLock l(&mutex_);
-      WriteBufferManagerStallWrites();
+  for (size_t i = 0; i < write_buffer_manager_.size(); i++) {
+    if (UNLIKELY(status.ok() && write_buffer_manager_[i]->ShouldStall())) {
+      if (write_options.no_slowdown) {
+        status = Status::Incomplete("Write stall");
+      } else {
+        InstrumentedMutexLock l(&mutex_);
+        WriteBufferManagerStallWrites(i);
```

Comment:
For simplicity, we don't need to allow stalling from cf level wbm. For default and write cf, we will still use DB level wbm and reuse the old code path.

Reply:
Good suggestion and done.